srcpkgs/python3-3to2/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'python3-3to2'
+pkgname=python3-3to2
+version=1.1.1
+revision=1
+wrksrc="3to2-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools unzip"
+makedepends="python3-devel"
+depends="python3 python3-setuptools unzip"
```

Comment:
I think it is. Because almost every package which has `python-setuptools` as hostmakedepends does it have as depends too. E.g.: `python-zope.component`